tests/sentry/utils/test_urllib3_timeout.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+import mock
+import pytest
+
+from urllib3 import HTTPConnectionPool
+from urllib3.exceptions import HTTPError, ReadTimeoutError
+
+from sentry.utils.snuba import RetrySkipTimeout
+
+
+class FakeConnectionPool(HTTPConnectionPool):
+    def __init__(self, connection, **kwargs):
+        self.connection = connection
+        super(FakeConnectionPool, self).__init__(**kwargs)
+
+    def _new_conn(self):
+        return self.connection
+
+
+def test_retries():
+    """
+    Tests that, even if I set up 5 retries, there is only one request
+    made since it times out.
+    """
+    conneciton_mock = mock.Mock()
```

Comment:
```suggestion
    connection_mock = mock.Mock()
```

Reply:
I need to find a way to formally introduce the word `connecIton` in the english vocabulary